src/reducers/reset.test.ts

```diff
@@ -0,0 +1,83 @@
+import {combineReducers} from 'redux';
+import {RESET} from '../actions';
+import {DEFAULT_PERSISTENT_STATE, PersistentState} from '../models/index';
+import {makeResetReducer, ResetIndex} from './reset';
+
+describe(RESET, () => {
+
+  const persistentStateToReset: ResetIndex<PersistentState> = {
+    bookmark: true,
+    config: false,
+    log: false,
+    shelfPreview: true
+  };
+
+  const getResetReducer = (defaultState = DEFAULT_PERSISTENT_STATE) => {
```

Comment:
Maybe just export persistentReducer in `index.ts` and use it?

Reply:
Since its a unit test for reset reducer just wanted it to be self contained. If we change `persistentReducer` this unit test should be independent of that change as it is testing only reset reducer. May be if I change the model then it can become totally independent.